src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3416,6 +3421,147 @@ namespace ZoneTempPredictorCorrector {
             }
         }
 
+        // Setpoint revision for onoff thermostat
+        Real64 TempTole = 0.02;
+        Real64 Tprev;
```

Comment:
These two lines can move inside the main if block below.  No need for these to be created if there aren't any NumOnOffCtrZone.

Reply:
@mjwitte Yes. I move these two line inside the if block.